### DISCUSSION TITLE
`just install` fails with `sh: line 1: ./common/scripts/package: No such file or directory`

### DISCUSSION THREAD

**Discussion by Niedzwiedzw** [cetz-package/cetz#642] [Q&A] [answered]

**Comment by johannes-wolf** [comment, marked as answer]
Maybe you forgot to fetch the git submodules?
`git submodule update --init`

**Reply by josephleblanc** [reply]
I just tried to install this in the same way. This step is unusual to me, as I have never needed to do so when cloning or installing other github projects.

Is this somewhere in the installation instructions and I just missed it?

**Reply by johannes-wolf** [reply]
Normal procedure for repositories with sub-modules. I can add a note to the Readme.
